### PR TITLE
Updates to Google Sign-In plugin for codelab

### PR DESCRIPTION
**Pull request by collinjackson**
Still in progress:
* Android updates
* Test

### CODE REVIEW

**Comment by collinjackson** [comment]
@efortuna

File: lib/google_sign_in.dart [hunk 1]
```diff
@@ -71,54 +92,36 @@ class GoogleSignIn {
...
-  static void initialize({
-    List<String> scopes,
-    String hostedDomain,
-  }) {
-    assert(_instance == null);
-    _scopes = scopes;
-    _hostedDomain = hostedDomain;
-  }
+  GoogleSignIn({ this.scopes, this.hostedDomain })
+    : _channel = const MethodChannel('plugins.flutter.io/google_sign_in'),
```
Comment (efortuna) [inline lib/google_sign_in.dart]: this seems like it should end in a ; but maybe the git diff is confusing here.

File: lib/google_sign_in.dart [hunk 2]
```diff
@@ -171,13 +174,14 @@ class GoogleUserCircleAvatar extends StatelessWidget {
 
   Widget _buildClippedImage(BuildContext context, BoxConstraints constraints) {
     assert(constraints.maxWidth == constraints.maxHeight);
+    String url = _sizedProfileImageUrl(
+      MediaQuery.of(context).devicePixelRatio * constraints.maxWidth,
+    );
```
Comment (efortuna) [inline lib/google_sign_in.dart]: this formatting seems odd, but you all may have a different style than the Dart team.

Comment (collinjackson) [inline lib/google_sign_in.dart]: I think it's consistent with https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters

It won't fit onto a single line so I had to break it somewhere.

**Comment by collinjackson** [comment]
Done. Pushed some TBR fixes for the Android build, I'll try to get a core Flutter team to review this again once it's more documented.

File: lib/google_sign_in.dart (hunk 2 above)
Comment (efortuna) [inline lib/google_sign_in.dart]: Yeah that trailing paren is what strikes me as odd, but looks like you guys do it all over the place so I'll be quiet. :-)

**Comment by collinjackson** [comment]
Trailing parens are the future! The "flutter create" template app extols their virtues in a comment in main.dart :)

**Pull request merged**